### PR TITLE
Add rules for OCP-25868

### PR DESCRIPTION
**Pull request by xiaocwan**
@yanpzhan 
Please review, thanks!

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -48,9 +48,14 @@ check_button_enabled:
     selector:
       xpath: //button[contains(., '<button_text>') and not(@disabled)]
 check_text_not_a_link:
+  element: 
+    selector:
+      xpath: //*[contains(@class,'co-resource-item__resource-name') and text()='<text>' and not(@href)]
+click_text_not_a_link:
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Where do we need this rule?

File: lib/rules/web/admin_console/4.3/logs.xyaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# logs page
+switch_to_second_container:
+  elements:
+  - selector:
+      xpath: //button[contains(@class,'pf-c-dropdown__toggle')]//span[contains(text(),'<current_container_name>')]
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.3/logs.xyaml]: Suggest to use switch_to_other_container as rule name.
 And no need to pass <current_container_name> in xpath, try to use:
xpath: //button[@data-test-id='dropdown-button']

File: lib/rules/web/admin_console/4.3/logs.xyaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    op: click
+  action: check_page_contains
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.3/logs.xyaml]: You didn't pass parameter in scenario for check_page_contains, and also this action is not a must.

File: lib/rules/web/admin_console/4.3/logs.xyaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  elements:
+  - selector:
+      xpath: //button[contains(@class,'pf-c-dropdown__menu-item')]//span[contains(text(),'<new_container_name>')]
+    op: click
```
Comment (yanpzhan) [inline lib/rules/web/admin_console/4.3/logs.xyaml]: This xpath could be replaced by existing rule: click_one_item

File: lib/rules/web/admin_console/4.3/logs.xyaml (hunk 3 above)
Comment (xiaocwan) [reply lib/rules/web/admin_console/4.3/logs.xyaml]: will remove

**Comment by yanpzhan** [comment]
LGTM

**Comment by xiaocwan** [comment]
@pruan-rht Could you help to merge it plz when you have time? Thanks!

**Pull request merged**